ddl/backfilling_dist_scheduler.go

```diff
@@ -160,6 +163,12 @@ func (s *backfillDistScheduler) GetSubtaskExecutor(ctx context.Context, task *pr
 	}
 }
 
+func (s *backfillDistScheduler) IsIdempotent(*proto.Subtask) bool {
+	// unique index depends on MVCC to do deduplicate, so it's not idempotent.
+	// we might save engine TS later to make it idempotent.
+	return !s.uniqueIndex
```

Comment:
Currently, the unique index should also be idempotent.

Reply:
as we have discussed, will mark all add-index subtask as Idempotent